frontend/public/components/chargeback.tsx

```diff
@@ -377,18 +379,23 @@ const ReportsPage_: React.SFC<ReportsPageProps> = props => {
     </div>;
   }
   return <div>
-    <div className="co-well">
-      <h4>Getting Started</h4>
-      <p>
-      Chargeback is not yet installed and enabled.
-      See our documention for instructions on how to install Chargeback Report on your Tectonic Cluster.
-      </p>
-      <p>
-        Chargeback is an alpha feature.
-      </p>
-      <a href="https://coreos.com/tectonic/docs/latest/reports/install-chargeback.html" target="_blank" rel="noopener noreferrer">
-        <button className="btn btn-info">Installing Chargeback Report <ExternalLinkAltIcon /></button>
-      </a>
+    <div className="co-m-pane__body">
+      <HintBlock
+        title="Getting Started"
+        body={
+          <>
+            <p>
+              Chargeback is not yet installed and enabled.
+              See our documention for instructions on how to install Chargeback Report on your Tectonic Cluster.
+            </p>
+            <p>
+              Chargeback is an alpha feature.
+            </p>
+            <a href="https://coreos.com/tectonic/docs/latest/reports/install-chargeback.html" target="_blank" rel="noopener noreferrer">
```

Comment:
This doc is out of date as well. We should be pointing to the new topic in the OpenShift docs. I'm not sure if it exists yet.

cc @chancez

Reply:
Removed entire block per https://github.com/openshift/console/pull/2718#issuecomment-531862366